src/index.ts

```diff
@@ -43,11 +43,16 @@ export const functionValidator = (handlerFn: HandlerFn): HandlerFn => {
         }
 
         return validator(token, accountSid, authToken)
-            .then((result) => {
-                event.TokenResult = result;
-                return handlerFn(context, event, callback);
-            })
-            .catch(failedResponse);
+          .then((result) => {
+              event.TokenResult = result;
+              const promise = handlerFn(context, event, callback);
+              if (promise === undefined) {
```

Comment:
also check for null?

Reply:
Null is an acceptable response type. I'm checking whether the `handlerFn` is actually returning something or not.